### DISCUSSION TITLE
List dropdown icon off centre in Firefox

### DISCUSSION THREAD

**Discussion by peteryates** [community/community#9057] [Other Feature Feedback, Questions, & Ideas] [answered]
The new list dropdown icon doesn't look correctly aligned on my computer, it's off-centre both vertically and horizontally. 

I'm using Firefox 95 on Fedora Workstation 35.

![Screenshot from 2021-12-18 16-13-37](https://user-images.githubusercontent.com/128088/146647997-0524a665-5427-477c-b292-f541e92349b3.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report! This should be fixed, but let me know if you're still seeing the misalignment.

**Reply by peteryates** [reply]
Thanks @brianlovin - it's now centred horizontally but looks like it's too low vertically - I assume it's meant to be in line with the triangle in the 'Watch' button?

![Screenshot from 2022-01-05 09-07-26](https://user-images.githubusercontent.com/128088/148191651-3736de36-3d49-41c3-ab38-6eec8a5a22e2.png)

**Reply by brianlovin** [reply]
Yes that looks very low, but all of the other icons and text look misaligned in each of these buttons, so something else is going on here. I'm not able to reproduce this on Firefox + macOS, so it might have to do with the Fedora Workstation 35 setup (which I'm not familiar with).

Do you find that all other icons are misaligned across GitHub? If so, this might be systems-level issue that I can get to the right team to investigate 😊

**Reply by peteryates** [reply]
No, everything else across GitHub looks fine to me, other than these buttons/dropdowns I haven't noticed any other issues.

**Reply by brianlovin** [reply]
In the sidebar, how does this button look for you?

<img width="349" alt="Screen Shot 2022-01-05 at 10 47 30@2x" src="https://user-images.githubusercontent.com/1923260/148271901-861fe296-e367-44a8-ba3e-5ccb2bb4f906.png">

Or if you click into a notification, in the blue banner at the top do you see these buttons?

<img width="529" alt="Screen Shot 2022-01-05 at 10 47 43@2x" src="https://user-images.githubusercontent.com/1923260/148271931-17dae359-0f9e-4cde-8517-9ed8532a1879.png">

**Reply by peteryates** [reply]
Everything appears fine.

![Screenshot from 2022-01-06 10-24-37](https://user-images.githubusercontent.com/128088/148368508-b0d77c2d-74f6-49ef-92ad-2bfcbaf76196.png)
![Screenshot from 2022-01-06 10-24-30](https://user-images.githubusercontent.com/128088/148368511-56408875-9e66-4eb1-9532-5755956789b3.png)

**Reply by peteryates** [reply]
Just had a quick look and it appears to be the `.btn.sm .octicon { vertical-align: text-top; }` that's causing the problem.


https://user-images.githubusercontent.com/128088/148368824-b9fb4dc0-1450-4710-aa24-863ff6e022be.mp4

**Reply by brianlovin** [reply]
Thanks for the screenshots, I'll pass this along to our team. I can't reproduce the misalignment on any of my browsers on macOS, so it might be something to do with your specific setup. Appreciate it!

**Reply by peteryates** [reply]
Yeah - could be.

When I start Firefox in safe mode (no addons/plugins) it still happens. Perhaps it's just a Linux thing.

I'm on Firefox 95.0.2 using Fedora 35 with the latest GNOME Desktop.
